release/models/mpls/openconfig-mpls-ldp.yang

```diff
@@ -918,6 +942,32 @@ module openconfig-mpls-ldp {
     }
   }
 
+  grouping mpls-ldp-label-advertisement-config {
+    description
+      "Grouping containing configuration to enable Downstream on demand in a session.";
+    leaf downstream-on-deman-enabled {
+      type boolean;
+      default "false";
+      description
+        "'true' if session downstream on demand is enabled.";
```

Comment:
Please use complete sentences here -- i.e., `If this leaf is set to true, then downstream-on-demand is enabled for the neighbor.` or similar :-)

Reply:
check new description